### ISSUE TITLE
Consecutive Rotate calls on ImageMobject causes translation

### ISSUE THREAD

**Issue by Staticity** [ManimCommunity/manim#3067]
## Description of bug / unexpected behavior
If I rotate an ImageMobject by 60 degrees CW then 60 degrees CCW, it induces a translation despite being centered at (0, 0, 0) originally. The ImageMobject.get_center() is changed after the first rotation for some reason.

## Expected behavior
A rotation by +x then -x degrees should go back to identity

## How to reproduce the issue

<details><summary>Code for reproducing the problem</summary>

```py
`
class RotateBackForth(ThreeDScene):
    def construct(self):
        c = Circle()
        c.set_z_index(1)
        self.add(c)

        image = Square()
        self.add(image)
        self.play(FadeIn(image))
        self.play(Rotate(image, PI / 3))
        self.play(Rotate(image, -PI / 3))


class RotateBackForthImage(ThreeDScene):
    def construct(self):
        data = np.zeros((290, 290))
        data[:, 0:5] = 255
        data[0:5, :] = 255
        data[:, -6:-1] = 255
        data[-6:-1, :] = 255

        c = Circle()
        c.set_z_index(1)
        self.add(c)

        image = ImageMobject(data)
        self.add(image)
        self.play(FadeIn(image))
        self.play(Rotate(image, PI / 3))
        self.play(Rotate(image, -PI / 3))
`
```

</details>

## Additional media files

<details><summary>Images/GIFs</summary>

https://user-images.githubusercontent.com/4411336/206604126-87ad3f98-f6b1-46f9-bb5d-03c5432cf7af.mp4

https://user-images.githubusercontent.com/4411336/206604584-a48366d4-e974-43d8-a10b-c9e7b496486d.mp4
</details>

## System specifications

<details><summary>System Details</summary>

- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
- RAM:
- Python version (`python/py/python3 --version`):
- Installed modules (provide output from `pip list`):
```
- macOs Monterey (12.6.1)
- Python 3.8.9
```
</details>

**Comment by linuxr** [comment]
Hi , for the sample `RotateBackForthImage` , add some debug info, you will find the image's center would be changed after rotated.
```
        self.play(FadeIn(image))
        print(image.get_center())
        self.play(Rotate(image, PI / 3))
        print(image.get_center())
        self.play(Rotate(image, -PI / 3))
        print(image.get_center())
```

but in the first sample `RotateBackForth`, the image's center would not be changed after ratated.

**Comment by vaclavblazej** [comment]
discussed on [Discord](https://discord.com/channels/581738731934056449/1143479267372572792)

**Comment by behackl** [comment]
Resolved via #3340.
